service/controller/v14/resource/cloudformation/main_stack_test.go

```diff
@@ -444,6 +449,13 @@ func TestMainHostPostTemplateExistingFields(t *testing.T) {
 		fmt.Println(body)
 		t.Fatal("route header not found")
 	}
+
+	if !strings.Contains(body, "  GuestNSRecordSet:") {
+		fmt.Println(body)
+		t.Fatal("GuestNSRecordSet resource not found")
```

Comment:
Test would produce better output by using `t.Fatalf` and passing it the body as argument.

Reply:
copy/paste